### PR TITLE
RDISCROWD-3544 add email report to admins 

### PR DESCRIPTION
**Pull request by ywrsusan**
*Issue number of the reported bug or feature request: #<3544>*
https://jira.prod.bloomberg.com/browse/RDISCROWD-3544

**Testing performed**
Add unit test for new code;
Add logs for future debug after deployed

### CODE REVIEW

File: pybossa/default_settings.py [hunk 1]
```diff
@@ -108,6 +108,10 @@
 # Send emails weekly update every
 WEEKLY_UPDATE_STATS = 'Sunday'
 
+# Send emails weekly to admin
+WEEKLY_ADMIN_REPORTS = 'Monday'
+WEEKLY_ADMIN_REPORTS_EMAIL = ['tbarrett@bloomberg.net','aross107@bloomberg.net']
```
Comment (dchhabda) [inline pybossa/default_settings.py:113]: Please include this under settings_local. Thanks!

File: pybossa/jobs.py [hunk 2]
```diff
@@ -1175,9 +1200,11 @@ def push_notification(project_id, **kwargs):
                                filters=filters)
 
 
-def mail_project_report(info, current_user_email_addr):
+def mail_project_report(info, email_addr):
```
Comment (dchhabda) [inline pybossa/jobs.py:1203]: This is to send weekly project report. hence, can it be send_weekly_project_report similar to other weekly report functions. with changing this, loggging message would also need an update.

File: pybossa/jobs.py [hunk 3]
```diff
@@ -1445,6 +1472,12 @@ def write_user(writer, user):
     def add_headers(writer):
         writer.writerow(exportable_attributes)
 
+    if isinstance(email_addr, list):
+        recipients = email_addr
+        current_app.logger.info(u'Scheduling export_all_users job send to admins')
```
Comment (dchhabda) [inline pybossa/jobs.py:1477]: logging can be outside of if else?

File: pybossa/jobs.py (hunk 2 above)
Comment (ywrsusan) [reply pybossa/jobs.py:1203]: this func is also executed when user requests project report on UI.

File: pybossa/jobs.py [hunk 4]
```diff
@@ -1004,6 +1006,29 @@ def notify_task_progress(info, email_addr, queue='high'):
                 queue=queue)
     enqueue_job(job)
 
+def get_weekly_admin_report_jobs():
+    """Return email jobs with weekly report to admins"""
+    send_emails_date = current_app.config.get('WEEKLY_ADMIN_REPORTS')
+    recipients = current_app.config.get('WEEKLY_ADMIN_REPORTS_EMAIL')
```
[the remark this answers is not in the document]
Comment (ywrsusan) [reply pybossa/jobs.py:1012]: yes, updated.

File: pybossa/jobs.py (hunk 2 above)
Comment (dchhabda) [reply pybossa/jobs.py:1203]: gotcha, thanks

File: pybossa/jobs.py (hunk 3 above)
Comment (ywrsusan) [reply pybossa/jobs.py:1478]: this function is also reused when user requests user report from GIGwork UI. I wanted to make the logger msg more clear about either it's from weekly job or from users' request, so i make the logger msg inside IF.

Comment (dchhabda) [reply pybossa/jobs.py]: seems little hacky to determine its weekly or user report using variable type here. Does it make sense to set logs under caller func instead?

**Review by dchhabda — APPROVED** [review]

File: pybossa/jobs.py [hunk 5]
```diff
@@ -1004,6 +1006,29 @@ def notify_task_progress(info, email_addr, queue='high'):
...
     enqueue_job(job)
 
+def get_weekly_admin_report_jobs():
+    """Return email jobs with weekly report to admins"""
+    send_emails_date = current_app.config.get('WEEKLY_ADMIN_REPORTS').lower()
+    recipients = current_app.config.get('WEEKLY_ADMIN_REPORTS_EMAIL')
+    today = datetime.today().strftime('%A').lower()
+    timeout = current_app.config.get('TIMEOUT')
+    if recipients and today == send_emails_date:
+        info = dict(timestamp=datetime.datetime.now().isoformat(),
```
Comment (XiChenn) [inline pybossa/jobs.py:1016]: This should be datetime.now().isoformat() as import is "from datetime import datetime"

**Pull request merged**